minimalscreenshotter.ts

```diff
@@ -0,0 +1,65 @@
+import EnvironmentConfig from './src/EnvironmentConfig';
+import { remote } from 'webdriverio';
+import { BrowserFactory, DEFAULT_CONNECTION_PARAMS } from './src/TBBrowserFactory.js';
+import {
+	RequestedStandaloneCapabilities,
+	WebdriverIOConfig,
+} from '@wdio/types/build/Capabilities';
+import { promises as fs } from 'fs';
+
+/**
+ * This minimal screenshotting tool allows to test making single screenshots on testingbot
+ */
```

Comment:
[Nitpick] Could you pull this comment to the top of the file? This is where I'd expect a description of what the file is about